layout-server/src/main/java/org/finos/vuu/layoutserver/model/Layout.java

```diff
@@ -0,0 +1,29 @@
+package org.finos.vuu.layoutserver.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
```

Comment:
Is the no-args constructor needed for the model mapper?

Reply:
I'll double check.

Reply:
Removed it 🙂